grpc-sys/Cargo.toml

```diff
@@ -11,8 +11,21 @@ description = "FFI bindings to gRPC c core library"
 categories = ["external-ffi-bindings", "network-programming"]
 build = "build.rs"
 exclude = [
+    "grpc/doc/*",
```

Comment:
Yes. But these directories only occupy small spaces but requiring a long list to exclude. And the project can't compile without including `src/csharp`, although we don't use any `src/csharp` code here.